css/index.css

```
@@ -1,3 +1,259 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+html {
+  margin: 100px;
+  margin-top: 25px;
+}
+
+a {
+  color: gray;
+  text-decoration: none;
+  text-shadow: 2px 2px 5px gray;
+}
+
+nav {
+  width: 800px;
+  width: 100%;
+  height: 80px;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.logo {
+  display: flex;
+  align-self: flex-start;
+}
+
+h1 {
+  font-size: 60px;
+  text-align: center;
+  text-shadow: 2px 2px 5px gray;
+  display: flex;
+}
+
+.code-img {
+  margin-bottom: 50px;
+}
+
+.jumbo {
+  display: flex;
+  justify-content: space-around;
+}
+
+.jumbo-copy {
+  text-align: center;
+  margin-top: 60px;
+}
+
+body{
+  max-width: 800px;
+  margin: auto;
+}
+
+button{
+  border: solid black;
+  border-radius: 12px;
+}
+
+button:hover {
+  background-color: #4CAF50; 
+  color: white;
+}
+
+.top-p {
+  font-family: 'Source Sans Pro', sans-serif;
+  display: flex;
+  margin-bottom: 20px;
+  margin-top: 15px;
+}
+
+.middle-img {
+  width: 100%;
+}
+
+.bottom-p {
+  font-family: 'Source Sans Pro', sans-serif;
+  display: flex;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.contact {
+  width: 20%
+}
+
+contact > p {
+  line-height: 2.0;
+}
+
+.copyr{
```

```suggestion
.copy-r{
```

---

best practices naming convention for classes is to use hyphens and avoid overly ambiguous names
```suggestion
.copyright {
```